test/recipes/80-test_ssl_old.t

```diff
@@ -456,7 +456,7 @@ sub testssl {
             # the default choice if TLSv1.3 enabled
             my $flag = $protocol eq "-tls1_3" ? "" : $protocol;
             foreach my $cipher (@{$ciphersuites{$protocol}}) {
-                if ($protocol eq "-ssl3" && $cipher =~ /ECDH/ ) {
+                if (($protocol eq "-ssl3" || $no_ec) && $cipher =~ /ECDH/ ) {
```

Comment:
This is not the right place to fix this.  Take a look above at line 414, and see what changes need to be made to the exclusions.

Reply:
The correct change is I think `s/kECDH/kECDHE/` on line 412.

Reply:
Oh, interesting...
Can you explain why
`openssl ciphers -s -tls1_2 ALL`
outputs ECDHE-ARIA256-GCM-SHA384 and ECDHE-ARIA128-GCM-SHA256
when configured with no-ec ???

Reply:
could it be that this is the root cause here ?
./config no-ec
=> openssl ciphers  ALL |sed 's/:/\n/g'|grep EC
ECDHE-ARIA256-GCM-SHA384
ECDHE-ARIA128-GCM-SHA256

Reply:
Yes, @bernd-edlinger I believe you're right.  Paging @paulidale

Reply:
how is the string "-kECDHe" in line 399 supposed to work?
from grep it looks like not implemented:

```
grep -r kECDHe *
CHANGES:         kECDHe   - ECDH cert, signed with ECDSA
include/openssl/ssl.h:# define SSL_TXT_kECDHe          "kECDHe"
test/recipes/80-test_ssl_old.t:        my $ciphers = "-EXP:-PSK:-SRP:-kDH:-kECDHe";
```